scipy/__init__.py

```diff
@@ -66,7 +66,7 @@
 
 from scipy._lib import _pep440
 # In maintenance branch, change to np_maxversion N+3 if numpy is at N
-np_minversion = '1.22.4'
+np_minversion = '2.0.0rc1'
```

Comment:
This is not correct, it would ban imports of NumPy prior to the 2.x series, which we still allow.

Reply:
I realize this is a serious error, sorry! Also I found PR#20459 just after committing.. I have pushed a fix to the problem you report, and a few changes which should resolve the pytest version guard problem. Sorry for the duplicate and, again, for the unfortunate error!